source/acquaman/AMGenericContinuousScanController.cpp

```diff
@@ -101,6 +101,9 @@ void AMGenericContinuousScanController::buildScanControllerImplementation()
 
 		foreach (AMRegionOfInterest *region, configuration_->regionsOfInterest()){
 
+			// This is sufficient to add a region of interest on all detectors as they should by synchronized via AMBeamline::synchronizeXRFDetectors.
+			detector->addRegionOfInterest(region);
```

Comment:
@helfrij @dretrex Yes, I just moved this LOC from Ln113 to here to make it clear that we added the ROI first